packages/common/src/Environments.ts

```diff
@@ -27,13 +27,15 @@ const defaultPenalizerConfiguration: PenalizerConfiguration = {
 }
 
 const defaultRelayHubConfiguration: RelayHubConfiguration = {
-  gasOverhead: 54771,
-  postOverhead: 19073,
+  gasOverhead: 61976,
```

Comment:
notice that devFee just added 7200 gas to the request.
(can be somewhat improved, by arranging fields into same-slot. e.g.  devFee and devAddress. need to be added when entire config is optimized)